resources/assets/js/views/user/UserListing.vue

```diff
@@ -673,7 +676,7 @@ const statusValue = ref();
 
 const statusUsername = ref();
 const deleteUsername = ref();
-
+const loader = ref(false);
```

Comment:
set loader true during initialization if it is to be set true at the beginning of onmounted